doc/articles/platform-specific-csharp.md

```diff
@@ -68,7 +68,7 @@ Heavy usage of `#if` conditionals makes code hard to read and comprehend. A bett
 
 ### A simple example
 
-Shared code in `PROJECTNAME.Shared/NativeWrapperControl.cs`:
+Shared code in `PROJECTNAME/NativeWrapperControl.cs`:
```

Comment:
> Previously, when we used shared projects, the code file in `PROJECTNAME.Shared` and the one in PROJECTNAME.SpecificPlatform` were to be compiled together in the same compilation, producing a single asssembly.

Shared projects created one binary per head, and we now have one binary per target framework for the new library project.

> @jeromelaban to confirm, and to advise on how to move forward. My suggestion would be to delete the "Partial class definitions" approach in this doc page.

Partial definitions are still valid, even if's not yet possible to make a distinction between wasm and skia. It will be possible in the future, likely in the next net8 preview with the introduction of `net8-browser`. 

We actually need to modify the next block to be:

---

Platform-specific code in `PROJECTNAME/NativeWrapperControl.Android.cs`:
```csharp
#if __ANDROID__
public partial class NativeWrapperControl : Control {
...
		private View CreateNativeView() {
			... //Android-specific code
```

Reply:
The article considers conditionals in a shared file and placing the code in the desired platform head as 2 separate approaches. As far as I currently understand it, partial definitions only work in the class library and should also be guarded by if conditionals, so these are similar to the first approach. Since the class library is compiled once per target framework (separate from each head), placing "the code in a file which is only included in the desired platform head" doesn't work anymore.

I've made more additions to fix/clarify the distinction.